includes/settings/js/src/components/ContentModelDropdown.jsx

```diff
@@ -58,6 +58,13 @@ export const ContentModelDropdown = ({ model }) => {
 				<Icon type="options" />
 			</button>
 			<div className={`dropdown-content ${dropdownOpen ? "" : "hidden"}`}>
+				<a
+					className="new"
+					href={`/wp-admin/post-new.php?post_type=${slug}`}
```

Comment:
Since this takes the user away from the current screen, adding this will make it consistent with how the "Open in GraphiQL" link works, which also takes the user away from the current screen.

```suggestion
					href={`/wp-admin/post-new.php?post_type=${slug}`}
					target="_blank"
					rel="noopener noreferrer"
```